### ISSUE TITLE
Linux 32bit, Arm7l and Arm64 builds

### ISSUE THREAD

**Issue by coreybruce** [ruffle-rs/ruffle#7197]
**Describe the feature**
Please do compiled builds for Linux 32bit, Arm7l and Arm64 for users to natively download for those architectures as they are missing.

**Comment by coreybruce** [comment]
> Note: I believe Ruffle still requires Vulkan drivers for the rendering on Linux. Those aren't as common for the ARM world.
> 
> Would still be great to have as builds though.

No it has the options to use both OpenGL and Vulkan and yes I believe Vulkan is on Arm

**Comment by parkerlreed** [comment]
Oh nice. Last I checked gl still wasn't active. Does appear to be working now!

**Comment by ActionWavele** [comment]
yes, this would be really useful, +1

**Comment by coreybruce** [comment]
Just tested nightly-2023-04-04 yesterday on my Raspberry pi 4 (Arm64) and it runs great with no issues. Please do a Arm64 compile!

**Comment by sodscg** [comment]
is that true? wow!
you should make a video!

**Comment by coreybruce** [comment]
Yeah obviously it still has the limitations of what flash games it supports as Ruffle is working on it but I tested a few games I know that work like dadnme, hobo 1, defend the castle and various other ones of that age and older and they work.

I could do a video tho I would have to figure out the best way to do so since it's on my pi 4

**Comment by kjarosh** [comment]
Aarch64 binaries are now available on Flathub: https://flathub.org/apps/rs.ruffle.Ruffle

**Comment by coreybruce** [comment]
> Aarch64 binaries are now available on Flathub: https://flathub.org/apps/rs.ruffle.Ruffle

Nice, you can also get the Arm64 package on the AUR package I maintain, I also compile the binaries :D 

https://aur.archlinux.org/packages/ruffle-bin

**Comment by kjarosh** [comment]
Aarch64 binaries are now available for nightly releases: https://github.com/ruffle-rs/ruffle/releases

They can be downloaded from our website too: https://ruffle.rs/downloads

**Comment by kjarosh** [comment]
i686 binaries are now available as well:

https://github.com/ruffle-rs/ruffle/releases
https://ruffle.rs/downloads
